### PR TITLE
Add Logitech touchpads, sort vendors alphabetically

### PR DESCRIPTION
**Pull request by RussianNeuroMancer**
Add Logitech touchpads, such as Logitech Wireless Touchpad, Logitech T650 and others.
Sort vendors alphabetically.

### CODE REVIEW

**Comment by whot** [comment]
travis CI says `Cannot parse /build/hwdb/70-touchpad.hwdb: Expected stringEnd (at char 1479), (line:45, col:1)` except that I really can't see what the issue here is. wait - I think you have a single space in the empty lines 52/58, that would be the parser failure.

Can you make sure this passes locally with `ninja -C builddir test` and fix whatever is going on there?

File: hwdb/70-touchpad.hwdb [hunk 1]
```diff
@@ -56,3 +50,14 @@ touchpad:usb:v05acp030e:*
 touchpad:usb:v05acp0265:*
  ID_INPUT_TOUCHPAD_INTEGRATION=external
  
+###########################################################
+# Logitech
+###########################################################
+touchpad:usb:v046d*
+ ID_INPUT_TOUCHPAD_INTEGRATION=external
+ 
+ ###########################################################
```
Comment (yuwata) [inline hwdb/70-touchpad.hwdb]: Unnecessary white space.

Comment (RussianNeuroMancer) [reply hwdb/70-touchpad.hwdb]: Please check new version.

File: hwdb/70-touchpad.hwdb [hunk 2]
```diff
@@ -56,3 +50,14 @@ touchpad:usb:v05acp030e:*
 touchpad:usb:v05acp0265:*
  ID_INPUT_TOUCHPAD_INTEGRATION=external
```
Comment (yuwata) [inline hwdb/70-touchpad.hwdb]: Please drop the white space.

File: hwdb/70-touchpad.hwdb [hunk 3]
```diff
@@ -56,3 +50,14 @@ touchpad:usb:v05acp030e:*
...
  
+###########################################################
+# Logitech
+###########################################################
+touchpad:usb:v046d*
+ ID_INPUT_TOUCHPAD_INTEGRATION=external
+
```
Comment (yuwata) [inline hwdb/70-touchpad.hwdb]: Also here. Please drop the white space.

File: hwdb/70-touchpad.hwdb (hunk 2 above)
Comment (RussianNeuroMancer) [reply hwdb/70-touchpad.hwdb]: Thanks for checking patch

File: hwdb/70-touchpad.hwdb (hunk 3 above)
Comment (RussianNeuroMancer) [reply hwdb/70-touchpad.hwdb]: Done

**Pull request merged**